File: app/src/main/java/com/example/mymultilanguagedemo/modulejava/demo/BaseActivity.java
package com.example.mymultilanguagedemo.modulejava.demo;

import android.content.Context;
import android.os.Bundle;


import androidx.appcompat.app.AppCompatActivity;

import com.example.mymultilanguagedemo.modulejava.lib.MultiLanguageUtil;

public class BaseActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }
    @Override
    protected void attachBaseContext(Context newBase) {
        super.attachBaseContext(MultiLanguageUtil.attachBaseContext(newBase));
    }
}

File: app/src/main/java/com/example/mymultilanguagedemo/kotlinmodule/demo/BaseActivity.kt
package com.example.mymultilanguagedemo.kotlinmodule.demo

import android.content.Context
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.mymultilanguagedemo.kotlinmodule.lib.MultiLanguageUtil

open class BaseActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun attachBaseContext(newBase: Context) {
        super.attachBaseContext(MultiLanguageUtil.attachBaseContext(newBase))
    }
}